test/ziggurat/fixtures.clj

```diff
@@ -145,7 +145,7 @@
     (mount-config)
     (mount-tracer)
     (mount-producer)
-    (binding [*bootstrap-servers* "localhost:9092"]
+    (binding [*bootstrap-servers* (get-in (config/ziggurat-config) [:stream-router :default :bootstrap-servers])]
```

Comment:
Good one.

Reply:
Make the same change here as well
```
(base) ➜  ziggurat git:(kafka-cluster-setup) ✗ ag "localhost" test
test/ziggurat/util/map_test.clj
19:    (let [a {:endpoint {:host "localhost" :port "3000"}}
21:          expected-map {:endpoint {:host "localhost" :port "3000"}}]

test/ziggurat/producer_test.clj
19:                   :bootstrap-servers      "localhost:8000"})
23:                           :bootstrap-servers    "localhost:9092"

test/ziggurat/dropwizard_metrics_wrapper_test.clj
49:  (let [statsd-config {:host "localhost" :port 8125 :enabled true}]

test/ziggurat/server/test_utils.clj
10:  (str "http://localhost:" port path))

test/ziggurat/clj_statsd_metrics_wrapper_test.clj
6:(def passed-host "localhost")
```

Reply:
A few references have been removed. Few others like in map and utils are pure logic tests. No point changing in those test cases.